Automattic/vip
# Add Slowlogs command

## Description

Customers could use the CLI to list Slow Query Logs as a similar feature used on the VIP Dashboard.

## Steps to Test

1. Check out PR.
1. Run `npm run build` and `npm link`
1. Run `node ./dist/bin/vip @SITE_ID.production slowlogs`
1. Verify list of slowlogs or "No logs found" if don't have any slowlogs to be listed
